src/main/java/uk/gov/hmcts/reform/em/npa/rest/RedactionResource.java

```diff
@@ -95,4 +106,37 @@ public ResponseEntity<Object> save(HttpServletRequest request,
                     .body(e.getMessage());
         }
     }
+
+    @Operation(
+        summary = "Delete all redactions for a document",
+        description = "Deletes all redactions associated with the provided DocumentId",
+        parameters = {
+            @Parameter(in = ParameterIn.HEADER, name = "authorization",
+                    schema = @Schema(type = "string"), required = true),
+            @Parameter(in = ParameterIn.HEADER, name = "serviceauthorization",
+                    schema = @Schema(type = "string"), required = true)
+        }
+    )
+    @ApiResponses({
+        @ApiResponse(responseCode = "204", description = "Deleted"),
+        @ApiResponse(responseCode = "400", description = "Invalid request"),
+        @ApiResponse(responseCode = "401", description = "Unauthorised"),
+        @ApiResponse(responseCode = "403", description = "Forbidden"),
+        @ApiResponse(responseCode = "500", description = "Server Error")
+    })
+    @DeleteMapping("/redaction/document/{documentId}")
+    public ResponseEntity<Void> deleteByDocumentId(
+            @RequestHeader(value = "Authorization", required = true) String auth,
+            @RequestHeader(value = "ServiceAuthorization", required = true) String serviceAuth,
+            @PathVariable UUID documentId) {
+        if (!deleteEnabled) {
+            throw new AccessDeniedException("Delete endpoint is disabled");
+        }
+        final long startMs = System.currentTimeMillis();
```

Comment:
Could you please change this to use below?
StopWatch stopWatch = new StopWatch();
        stopWatch.start();
stopWatch.stop();
        log.info("Deletion of redactions for document with Id {}  took {} ms", documentId ,stopWatch.getDuration().toMillis());

This will ensure we follow same and consistent approach across all EM API's.